.github/workflows/update-contributors.yml

```diff
@@ -37,5 +30,5 @@ jobs:
         git config --global user.name 'Github Action'
         git config --global user.email 'action@github.com'
         git add -u
-        git commit -m "Update contributors dataset" -a || exit 0
+        git commit -m "Update contributors dataset" || exit 0
```

Comment:
Why is the Git command followed with `|| exit 0`? What if the commit operation fails?